packages/@okta/vuepress-theme-prose/components/LiveWidgetOktaWidget.vue

```diff
@@ -63,7 +61,13 @@ export default {
           }
         });
       this.widget.renderEl({ el: '#widget-container' });
+    },
+    destroyWidget(){
+       if (this.widget) {
+      this.widget.remove();
     }
   }
```

Comment:
Can you auto-format or manually update the indentations here?

Reply:
I will kill my VS Code with fire. Thanks.